src/parser/tests/parser_tests.rs

```diff
@@ -1168,3 +1168,27 @@ r#"Variable {
     assert_eq!(expected, format!("{:#?}", x).as_str());
 }
 
+#[test]
+fn string_variable_declaration_can_be_parsed(){
```

Comment:
We're also missing sized string tests on the parser, or at least i seem to have missed them.

Reply:
added parser test in new commit